index.html

```diff
@@ -9025,6 +9039,263 @@ <h3><dfn>Take Element Screenshot</dfn></h3>
+
+ <li><p>If <var>background</var> is not a <a>Boolean</a>
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>Let <var>page</var> be the result of <a>getting a property
+  with default</a> named <code>page</code> and with a default of an
+  empty <a>Object</a> from the <var>parameters</var> argument.
+
+ <li><p>Let <var>pageWidth</var> be the result of <a>getting a
+  property with default</a> named <code>width</code> and with a
+  default of <code>21.59</code> from <var>page</var>.
+
+ <li><p>Let <var>pageHeight</var> be the result of <a>getting a
+  property with default</a> named <code>width</code> and with a
+  default of <code>27.94</code> from <var>page</var>.
+
+ <li><p>If either of <var>pageWidth</var> or <var>pageHeight</var> is
+ not a <a>Number</a>, or is less then 0, return <a>error</a>
+ with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>Let <var>margin</var> be the result of <a>getting a property
+  with default</a> named <code>margin</code> and with a default of an
+  empty <a>Object</a> from the <var>parameters</var> argument.
+
+ <li><p>Let <var>marginTop</var> be the result of <a>getting a
+  property with default</a> named <code>top</code> and with a
+  default of <code>1</code> from <var>margin</var>.
+
+ <li><p>Let <var>marginBottom</var> be the result of <a>getting a
+  property with default</a> named <code>bottom</code> and with a
+  default of <code>1</code> from <var>margin</var>.
+
+ <li><p>Let <var>marginLeft</var> be the result of <a>getting a
+  property with default</a> named <code>left</code> and with a
+  default of <code>1</code> from <var>margin</var>.
+
+ <li><p>Let <var>marginRight</var> be the result of <a>getting a
+  property with default</a> named <code>right</code> and with a
+  default of <code>1</code> from <var>margin</var>.
+
+ <li><p>If any
+ of <var>marginTop</var>, <var>marginBottom</var>, <var>marginLeft</var>,
+ or <var>marginRight</var> is not a <a>Number</a>, or is less then 0,
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>Let <var>shrinkToFit</var> be the result of <a>getting a
+  property with default</a> named <code>shrinkToFit</code> and with
+  default <code>true</code> from the <var>parameters</var> argument.
+
+ <li><p>If <var>shrinkToFit</var> is not a <a>Boolean</a>
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>Let <var>pageRanges</var> be the result of <a>getting a
+  property with default</a> named <code>pageRanges</code> from
+  the <var>parameters</var> argument with default of an
+  empty <var>Array</var>.
+
+ <li><p>If <var>pageRanges</var> is not an <a>Array</a>
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>When the user agent is next to <a>run the animation frame
+ callbacks</a>, let <var>pdfData</var> be the result of <a>trying</a>
+ to take UA-specific steps to generate a paginated representation of
```

Comment:
It seems to be implied below that `pdfData` is an array of PDFs, one per page, given removing pages; this should be spelt out here. (I mean a PDF itself can be paginated, so what does "a paginated representation" mean? Or have I misunderstood what the return value of this command is? Either way it should be clarified.

Reply:
There isn't supposed to be an arry of PDFs, just a single PDF. The point here was supposed to be that the PDF should use the appropriate CSS &c. for a paginated representation rather than something else .

Reply:
Can we clarify that it's a single multi-page PDF?